test/e2e/opm_test.go

```diff
@@ -405,6 +421,62 @@ var _ = Describe("opm", func() {
 			err = buildIndexWith(containerTool, indexImage, nextIndex, bundles[4:].images(), registry.ReplacesMode, true) // 1.0.1-overwrite
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		FIt("is consistent with existing update graphs", func() {
+			logger := logrus.NewEntry(logrus.StandardLogger())
+			tool := containertools.NewContainerTool(containerTool, containertools.NoneTool)
+			imageIndexer := indexer.ImageIndexer{
+				PullTool: tool,
+				Logger:   logger,
+			}
+			dbFile, err := imageIndexer.ExtractDatabase(".", redhatOperators, "", true)
+			Expect(err).NotTo(HaveOccurred(), "error extracting registry db")
+
+			db, err := sql.Open("sqlite3", fmt.Sprintf("file:%s", dbFile))
+			Expect(err).NotTo(HaveOccurred(), "Error reading db file")
+
+			querier := sqlite.NewSQLLiteQuerierFromDb(db)
+
+			graphLoader, err := sqlite.NewSQLGraphLoaderFromDB(db)
+			Expect(err).NotTo(HaveOccurred(), "Error reading db file")
+
+			packages, err := querier.ListPackages(context.TODO())
+			Expect(err).NotTo(HaveOccurred(), "Error listing packages")
+
+			var errs []error
+
+			adder := lregistry.NewRegistryAdder(logger)
+			for _, pkg := range packages {
+				existing, err := graphLoader.Generate(pkg)
+				Expect(err).NotTo(HaveOccurred(), "Error generating graph for package %s", pkg, existing)
+
+				for _, ch := range existing.Channels {
+					request := lregistry.AddToRegistryRequest{
+						Permissive:    false,
+						SkipTLS:       false,
+						InputDatabase: dbFile,
+						Bundles:       []string{ch.Head.BundlePath},
+						Mode:          registry.ReplacesMode,
+						ContainerTool: tool,
+						Overwrite:     true,
+					}
+
+					err = adder.AddToRegistry(request)
+					if err != nil {
+						errs = append(errs, fmt.Errorf("Error overwriting bundles for package %s: %s", pkg, err))
```

Comment:
@njhale looks like even if overwrite-latest fails we are just accumulating the errs here without raising them or doing anything with them.